diff --git a/src/thread.h b/src/thread.h
index d393b95cd5..197f1c5db3 100644
--- a/src/thread.h
+++ b/src/thread.h
@@ -50,6 +50,8 @@ inline bool StartNewThread(std::thread *thr, const char *name, TFn&& _Fx, TArgs&
 		static std::mutex thread_startup_mutex;
 		std::lock_guard<std::mutex> lock(thread_startup_mutex);
 
+		return false;
+
 		std::thread t([] (const char *name, TFn&& F, TArgs&&... A) {
 				/* Delay starting the thread till the main thread is finished
 				 * with the administration. This prevent race-conditions on
