diff --git a/cmake/FindICU.cmake b/cmake/FindICU.cmake
index d12f36b0a..a99b9eed0 100644
--- a/cmake/FindICU.cmake
+++ b/cmake/FindICU.cmake
@@ -31,7 +31,7 @@ This will define the following variables:
 
 find_package(PkgConfig QUIET)
 
-set(ICU_KNOWN_COMPONENTS "uc" "i18n" "le" "lx" "io")
+set(ICU_KNOWN_COMPONENTS "uc" "i18n" "le" "lx" "io" "data")
 
 foreach(MOD_NAME IN LISTS ICU_FIND_COMPONENTS)
     if(NOT MOD_NAME IN_LIST ICU_KNOWN_COMPONENTS)
